src/main/resources/templates/db_scripts/purchase_order_table.sql

```diff
@@ -1,2 +1,4 @@
 CREATE SEQUENCE IF NOT EXISTS ${myuniversity}_${mymodule}.po_number MAXVALUE 9999999999999999 START WITH 10000 CACHE 1 NO CYCLE;
 GRANT USAGE ON SEQUENCE ${myuniversity}_${mymodule}.po_number TO ${myuniversity}_${mymodule};
+
+CREATE UNIQUE INDEX purchase_order_po_number_idx ON ${myuniversity}_${mymodule}.purchase_order ((jsonb->>'po_number') );
```

Comment:
Please add `IF NOT EXISTS` to avoid problems during module upgrade

Reply:
Also, it seems like the convention is to name unique indexes with the `_idx_unique` or `_unique_idx` suffix